### PR TITLE
[3.1] Replace more uses of OS names with {EL}

### PR DESCRIPTION
**Pull request by ekohl**
(cherry picked from commit 434d771c89581a056faa6abc3fc3a67b3f9c4559)

### CODE REVIEW

File: guides/common/modules/proc_installing-the-satellite-server-packages.adoc [hunk 1]
```diff
@@ -4,48 +4,22 @@ ifdef::context[:parent-context: {context}]
...
+* xref:#el-8[{EL} 8]
+* xref:#el-7[{EL} 7]
 endif::[]
 
 .Procedure
 
 ifdef::foreman-el,katello,satellite[]
-== [[rhel-8]]{RHEL} 8
-:context: rhel8
+== [[rhel-8]][[el-8]]{EL} 8
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-the-satellite-server-packages.adoc:14]:
```suggestion
== [[el-8]]{EL} 8
```

File: guides/common/modules/ref_storage-requirements.adoc [hunk 2]
```diff
@@ -1,13 +1,9 @@
...
 = Storage Requirements
 
-ifdef::foreman-el,katello[]
-* xref:#storage-centos-7[CentOS 7]
-* xref:#storage-centos-8[CentOS 8]
-endif::[]
 ifdef::foreman-el,katello,satellite[]
-* xref:#storage-rhel-8[{RHEL} 8]
-* xref:#storage-rhel-7[{RHEL} 7]
+* xref:#storage-el-8[{EL} 8]
```
Comment (maximiliankolb) [inline guides/common/modules/ref_storage-requirements.adoc:5]: 👍

File: guides/common/modules/ref_storage-requirements.adoc [hunk 3]
```diff
@@ -17,58 +13,8 @@ ifdef::katello,satellite[]
...
-|/var/lib/pulp/ |1 MB |300 GB
-
-|/var/lib/qpidd/ |25 MB | Not Applicable
-endif::[]
-|====
-endif::[]
-
 ifdef::foreman-el,katello,satellite[]
-== [[storage-rhel-8]]{RHEL} 8
+== [[storage-rhel-8]][[storage-el-8]]{EL} 8
```
Comment (maximiliankolb) [inline guides/common/modules/ref_storage-requirements.adoc:17]:
```suggestion
== [[storage-el-8]]{EL} 8
```

File: guides/common/modules/ref_storage-requirements.adoc [hunk 4]
```diff
@@ -83,16 +29,14 @@ ifdef::foreman-el,katello,satellite[]
...
 |/var/lib/pulp/ |1 MB |300 GB
 
 |/var/lib/qpidd/ |25 MB | Not Applicable
 endif::[]
 |====
-endif::[]
 
-ifdef::foreman-el,katello,satellite[]
-== [[storage-rhel-7]]{RHEL} 7
+== [[storage-rhel-7]][[storage-el-7]]{EL} 7
```
Comment (maximiliankolb) [inline guides/common/modules/ref_storage-requirements.adoc:39]:
```suggestion
== [[storage-el-7]]{EL} 7
```

**Review by maximiliankolb — APPROVED** [review]
I am not sure yet if those "double anchors" break something downstream. I'd recommend not doing this, but won't block this PR because of it.

**Comment by ekohl** [comment]
> I am not sure yet if those "double anchors" break something downstream.

At least in upstream it generates `<a id="rhel-8"></a><a id="el-8"></a>` so I can't see that breaking. Not sure about other downstream tooling though.

> I'd recommend not doing this

I'd be OK with breaking the existing anchors. @Lennonka thoughts?

The impact will be that https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html/installing_satellite_server_in_a_connected_network_environment/installing_server_connected_satellite#rhel-8 breaks and becomes https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html/installing_satellite_server_in_a_connected_network_environment/installing_server_connected_satellite#el-8

Users who have bookmarked that will see the top of the page.

**Comment by Lennonka** [comment]
TBH I have no idea how that will behave. But it looks like it could make for an interesting circle of Hell. Let's do it!

**Pull request merged**